src/main/kotlin/com/team4099/robot2023/commands/drivetrain/FollowPathPlannerPathCommand.kt

```diff
@@ -121,6 +132,14 @@ class FollowPathPlannerPathCommand(val drivetrain: Drivetrain, val path: PathPla
   init {
     addRequirements(drivetrain)
 
+    if (useOdoFrame) {
+      odoTRobotSupplier = { drivetrain.odomTRobot }
+    } else {
+      odoTRobotSupplier = { drivetrain.fieldTRobot }
+      // if we're already in field frame we do not want to shift by `odoTField` again
+      shiftByOdoTField = false
+    }
```

Comment:
😬 don't do this, it will be extremely confusing later. If it's called `odoTRobotSupplier` it should only be supplying `odomTRobot` and not `fieldTRobot`. If you want it to change depending on what is selected call it `robotPoseSupplier` or something generic